apps/DemoApp/run-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+# Delete the old stuff
+rm -Rf Frameworks
+
+buildkite-agent artifact download "MUXSDKStats.framework.zip" . --step "buildkite.sh"
+unzip MUXSDKStats.framework.zip
+cd apps/DemoApp
+pod deintegrate && pod install
```

Comment:
Quick sanity check here, making sure I understand the steps for what buildkite is doing here:

1. clones
2. rebuild the frameworks (`buildkite.sh`)
3. open demoapp
4. install local version of the pod that just got built
5. open DemoApp workspace and run `test`

For step 4 to work correctly, it is important that the DemoApp `Podfile` is pointed to the local version of the pod, correct? (instead of cocoapods or a github branch, which we have done in the past).

Reply:
Yes that is correct - the Podfile must point to the local version of the `Mux-Stats-AVPlayer` pod for this to work as expected
```
pod 'Mux-Stats-AVPlayer', :path => '../..'
```
> instead of cocoapods or a github branch, which we have done in the past
hmm this is a good point 🤔   the tests would not be testing the SDK that we just built if this app points to a Github branch or Cocoapods for the AVPlayer SDK. But when would we do that with the intention of checking in the change?Because when it comes time to merge any of our branches, we'd always want DemoApp to point to the local version of the pod right?

Reply:
Makes sense, I don't think anything needs to be changed here.

> But when would we do that with the intention of checking in the change? Because when it comes time to merge any of our branches, we'd always want DemoApp to point to the local version of the pod right?

At one point we were pointing the DemoApp to Cocoapods so that if a developer looks around in this app they can easily copy/paste that line into their own app.

I think having the Cocoapods line commented out in the Podfile above this line is nice for that purpose, but overall I agree that it makes sense for the DemoApp to always be pointed to the local source

Reply:
Ahh I see - makes sense